integration/src/main/kotlin/kotlinx/benchmark/integration/ProjectBuilder.kt

```diff
@@ -31,35 +29,31 @@ benchmark {
     }
 }
 
-private val buildScript = run {
+private val kotlin_repo = System.getProperty("kotlin_repo_url").let {
+    if (it.isNullOrBlank())
+        "mavenCentral()"
+    else
+        "maven { url '$it' }"
+}
+
+private val buildScript =
     """
     buildscript {
+        repositories {
+            $kotlin_repo
+            maven { url '${System.getProperty("plugin_repo_url")}' }
```

Comment:
Do we get all required transitive dependencies if we override `kotlin_repo` to something other than `mavenCentral`?

Reply:
Now the mavenCentral repo is always in repositories list.